### PR TITLE
Fix FB-2814 "Users.requestUsernameFromID() doesn't work for yourself"

### PR DESCRIPTION
**Pull request by zfox23**
When calling requestUsernameFromID() from a script, the intended behavior is for it to work when you're requesting your own username even if you aren't an admin. This doesn't work before this fix.

**Test plan:**
* Log in to HiFi.
* With a friend, go into a domain where you **are not** an admin.
* Open the PAL.
* Verify that your username shows up below your display name in the PAL and that your friend's username **does not** appear below their display name.
* With a friend, go into a domain where you **are** an admin.
* Open the PAL.
* Verify that your username shows up below your display name in the PAL and that your friend's username **does** appear below their display name.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: e22c1ac30e35b376bfe28b0309536f9e1c5de911 available at: https://gist.github.com/34ce02faf8240f682ba326a75460937a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9397-e22c1ac.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9397-e22c1ac.dmg

**Comment by OmegaHeron** [comment]
This isn't working for me...

Test setup as follows;

Client 1 running this PR Interface and Sandbox on Windows 10 with full rights.
Client 2 running this PR Interface on Ubuntu Linux connecting to ^^ Sandbox *without* any admin rights.

Client 1 (has admin rights) can see its Display and User Name along with full data for any other agents present.

Client 2 (no admin rights) does not see its Username - only Display name (or anon if not set) and, as expected, sees on display name (or anon) for others.

Basically I see no change between this PR and master.

**Comment by zfox23** [comment]
Strange, @OmegaHeron... Just making sure...is Client 2 logged in?

I tested this with almost exactly the same setup as you and it worked as expected. Let me try again.

**Comment by OmegaHeron** [comment]
Yeah @zfox23, both machines have agents logged in. Checking one more time just to be sure I didn't screw up.

**Comment by OmegaHeron** [comment]
From Linux machine - client has no admin rights (and you can tell I'm logged in via user list caption)

hifi-snap-by-heez-zeeman-on-2017-01-16_15-30-04

From Windows machine - client has admin rights

hifi-snap-by-omegaheron-on-2017-01-16_15-29-45

From Windows machine in a domain where I have no admin rights

hifi-snap-by-omegaheron-on-2017-01-16_15-35-05

**Comment by zfox23** [comment]
@OmegaHeron: Very strange. I just tested this once more and it works fine. Is the version of Sandbox you're running based off of this PR? This is a domain-server change, not a client change.

Screenshot from client in domain without admin rights:
boz6hou

**Comment by OmegaHeron** [comment]
I tested against DS/AC provided with this PR - just checked, windows machine running stack PR9397, Interface 9397, linux box showing interface 9397 in title bar.  Bad merge or something otherwise screwy with build?

**Comment by OmegaHeron** [comment]
Reboot - uninstall/reinstall fixed my issues.

QA Pass

Acts as outline, general smoke test = good and regular PAL stuff unchanged.
